setup.py

```diff
@@ -152,6 +149,6 @@ def run(self):
 
 # perform the setup
 setup(
-        cmdclass=dict(build_ext=build_ext, install_data=install_data),
+        cmdclass={"build_ext": build_ext, "install": install},
```

Comment:
This change is spurious. Is there a reason why you want to get rid of the use of the dict() constructor?

Reply:
Right. But why not just change the key? That's easier to follow and if you feel strongly enough about the formatting you could change it separately -- along with other things you might not like. ;-)

Reply:
Revert the change. I'd like to contribute a "black" format if you're not against it.

Reply:
No objection. :-)